OctopusProjectBuilder.Uploader/Helpers/FakeEnvironmentRepository.cs

```diff
@@ -4,7 +4,7 @@
 using Octopus.Client.Model;
 using Octopus.Client.Repositories.Async;
 
-namespace OctopusProjectBuilder.Uploader.Tests.Helpers
```

Comment:
The project follows the rule where namespace corresponds the project & folder structure. 
Please revert the namespace changes.